json_schema/1.0.0.schema.json

```diff
@@ -0,0 +1,558 @@
+            "examples": [
+              "2020-01-01T12:00:00Z"
+            ]
+          },
+          "changelog": {
+            "type": "string",
+            "description": "A changelog for this version. May contain HTML."
+          },
+          "dependencies": {
+            "type": "array",
+            "description": "Required or optional dependencies for this version.",
+            "items": {
+              "$ref": "#/definitions/dependency"
+            }
+          },
+          "installation": {
+            "type": "object",
+            "description": "Installation Information for this version.",
+            "$ref": "#/definitions/installation"
+          },
+          "files": {
+            "type": "array",
+            "description": "An array containing the files for this version.",
+            "items": {
+              "type": "object",
+              "description": "A file object.",
+              "additionalProperties": false,
+              "required": [
+                "name",
+                "sha256",
+                "rel",
+                "downloads"
+              ],
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "The name of this file."
+                },
+                "sha256": {
+                  "type": "string",
+                  "description": "An SHA256 checksum of this file."
+                },
+                "rel": {
+                  "type": "string",
+                  "description": "The relation of this file.",
+                  "examples": [
+                    "primary",
+                    "source",
+                    "deobf",
+                    "javadoc",
+                    "installerWin",
+                    "installerMacos",
+                    "installerLinux",
+                    "installerGeneric",
+                    "versionJson",
+                    "compressedOverrides",
+                    "forgeUniversal",
+                    "forgeMdk"
+                  ]
+                },
+                "dependencies": {
+                  "type": "array",
+                  "description": "Dependencies for this specific file.",
+                  "items": {
+                    "$ref": "#/definitions/dependency"
+                  }
+                },
+                "installation": {
+                  "type": "object",
+                  "description": "Installation Information for this file.",
+                  "$ref": "#/definitions/installation"
+                },
+                "downloads": {
+                  "type": "array",
+                  "description": "An array containing download sources for this file.",
+                  "items": {
+                    "type": "string",
+                    "description": "A URL linking to this file.",
+                    "format": "uri"
+                  }
+                },   
```

Comment:
There is trailing whitespace in this line.

Reply:
Thanks, I will fix.